src/core/builtin_functions.cc

```diff
@@ -436,47 +436,121 @@ Value builtin_concat(Arguments arguments, const Location& /*loc*/)
   return std::move(result);
 }
 
+static Value warning( const std::shared_ptr<const Context>& context, const FunctionCall *call, const std::string & msg) {
+    LOG(message_group::Warning, call->location(), context->documentRoot(), msg.c_str());
+    return Value::undef(msg);
+}
+
+template <typename ...Args>
+std::string format(const std::string& format, Args && ...args)
+{
+    auto size = std::snprintf(nullptr, 0, format.c_str(), std::forward<Args>(args)...);
+    std::string output(size + 1, '\0');
+    std::snprintf(&output[0], size+1, format.c_str(), std::forward<Args>(args)...);
+    return output;
+}
+
+#define OBJECT_HELP "In an unnamed list, entries must be [key,value] to set or [key] to delete. The key must be <string>."
+
+static std::string builtin_object_unnamed(ObjectType & result, Value & value, int arg_index) {
+
+    std::string prior_args = std::string();
+        for ( int i=0; i<arg_index; i++) prior_args += "_, ";
+
+    switch(value.type()){
+    case Value::Type::OBJECT: {
+        const auto & obj = value.toObject();
+        for (const auto& key : obj.keys()) {
+            result.set(key, obj.get(key).clone());
+        }
+        return "";
+    }
+    case Value::Type::VECTOR: {
+        std::string prior_entries;
+
+        const VectorType & vector = value.toVector();
+        for (const auto& member : vector) {
+
+            if (member.type() != Value::Type::VECTOR) {
+                return format(
+            "object( %s[%s<%s>] ) Entry type is not a list, it is <%s>. " OBJECT_HELP, 
```

Comment:
This line, and many below, ends with a space.  I don't know whether OpenSCAD wants to avoid that, but at work we avoid it because some editors will eat that extra space and the result is spurious diffs.